meca:
  servo1_controller:
    type: "position_controllers/JointPositionController"
    joint: link1_joint1
    publish_rate: 50
      
  servo2_controller:
    type: "position_controllers/JointPositionController"
    joint: link2_joint1
    publish_rate: 50
      

